src/database/schemaBreakdown/field.ts

```diff
@@ -0,0 +1,47 @@
+import { Field, FieldPayload } from '../../model/field';
+import { Transaction } from 'knex';
+import { BreakDownRepository } from './breakdown';
+
+const TABLE_NAME = 'type_def_fields';
+const TABLE_COLUMNS = [
+	'name',
+	'description',
+	'is_nullable',
+	'is_array',
+	'is_array_nullable',
+	'is_deprecated',
+	'parent_type_id',
+	'children_type_id',
+];
+
+export class FieldTransactionRepository extends BreakDownRepository<
+	FieldPayload,
+	Field
+> {
+	private static instance: FieldTransactionRepository;
+
+	constructor() {
+		super(TABLE_NAME, TABLE_COLUMNS);
+	}
+
+	static getInstance(): FieldTransactionRepository {
+		if (!FieldTransactionRepository.instance) {
+			FieldTransactionRepository.instance =
+				new FieldTransactionRepository();
+		}
+
+		return FieldTransactionRepository.instance;
+	}
+
+	async insertIgnoreFields(trx: Transaction, data: FieldPayload[]) {
+		return super.insert(trx, data);
```

Comment:
I hope this is going to generate bulk insert? I'm worried about performance.. if I register schema that is 100 types * 20 properties.. how long the endpoint that take?

Reply:
Yes, it's for saving multiple at once. About performance, we can prepare some insights in the future.